tests/dataSources_test.go

```diff
@@ -857,7 +857,8 @@ var _ = Describe("DataSources", func() {
 				Eventually(func() bool {
 					cron := &cdiv1beta1.DataImportCron{}
 					Expect(apiClient.Get(ctx, dataImportCron.GetKey(), cron)).To(Succeed())
-
+					fmt.Println("dataImportCron " + dataImportCron.GetKey().String() + " status")
+					fmt.Printf("%#v \n", cron.Status)
```

Comment:
This will be very noisy in log.
Instead, can you add a `JustAfterEach()` block, with a call to `logObject()` function which is exactly for this use case:
https://github.com/kubevirt/ssp-operator/blob/da413a117962247d41c1134b4914643a189543dd/tests/misc_test.go#L360

Here is an example: https://github.com/kubevirt/ssp-operator/blob/da413a117962247d41c1134b4914643a189543dd/tests/misc_test.go#L224-L241

It uses `GinkgoWriter` which will only output to log if a test fails.
The function is currently local in `misc_test.go`, but you can move it to `tests_common_test.go`.

Reply:
If the tests will not fail, then there will be no record in log. It will log only in case of error. But your approach with JustAfterEach is better. I will update my PR.

Reply:
Ok, I thought that a regular `fmt.Printf()` always outputs, even when test is successful. Good to know.